config/packages/validator.yaml

```diff
@@ -1,3 +1,9 @@
 framework:
     validation:
+        enable_annotations: true
         email_validation_mode: html5
+
+        # Enables validator auto-mapping support.
+        # For instance, basic validation constraints will be inferred from Doctrine's metadata.
+        #auto_mapping:
+        #    App\Entity\: []
```

Comment:
let's uncomment this in anticipation with https://github.com/symfony/recipes/pull/664/files?